webapp/src/main/webapp/vue-apps/CallButton/components/JitsiMeetButton.vue

```diff
@@ -32,34 +37,59 @@ export default {
       settings: this.callSettings,
       log: null,
       callWindow: null
-      // callState: null
     };
   },
   computed: {
     callState: function() {
       return this.callSettings.callState;
     },
+    parentElement() {
+      if (this.$refs.jitsi) {
+        return this.$refs.jitsi.$el.parentElement.parentElement.parentElement.parentElement.parentElement;
+      }
+    },
     buttonTitle: function() {
       if (this.callState === "joined") {
+        if (this.$refs.jitsi && this.parentElement) {
+          return {
+            title: !this.parentElement.classList.contains("call-button-mini")
```

Comment:
Could we do this in single method (insetad of three) with provising a contextual label as param?

Reply:
It's extracted in the single method